src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -435,11 +449,8 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
             onClickAddGenesToHeatmap:()=>{
                 this.addHeatmapTracks(this.selectedHeatmapProfile, this.heatmapGeneInputValue.toUpperCase().trim().split(/\s+/));
             },
-            onClickRemoveHeatmap:action(()=>{
+            onClickRemoveHeatmap:action(() => {
                 this.molecularProfileIdToHeatmapTracks.clear();
-                if (this.sortMode.type === "heatmap") {
```

Comment:
whats going on here?

Reply:
For the reason I pointed out in the commit message for 499c12806ed72cd0415f30d8a3d2788112c6582d: there's no reason to uncluster the gene set heatmap when removing other heatmaps.

Reply:
but this way we'll lose the correct behavior when theres no geneset..

Reply:
oh maybe not because of the onRemove handler above, I see